reverb/cc/platform/default/build_rules.bzl

```diff
@@ -441,10 +447,15 @@ def reverb_pybind_extension(
             "-fexceptions",  # pybind relies on exceptions, required to compile.
             "-fvisibility=hidden",  # avoid pybind symbol clashes between DSOs.
         ],
-        linkopts = linkopts + _rpath_linkopts(module_name) + [
-            "-Wl,--version-script",
-            "$(location %s)" % version_script_file,
-        ],
+        linkopts = linkopts + _rpath_linkopts(module_name) +
+            select({"//reverb:macos": [
+                        "-Wl,-exported_symbols_list,$(location %s)" % exported_symbols_file,
```

Comment:
looks like you're doing it here; do you not need the exported symbols list above?

Reply:
yes, done in https://github.com/deepmind/reverb/pull/24/commits/32c3441d119217f2461b89b971d12acb87ee8c00